### PR TITLE
Trim whitespace for ProjectName

### PR DESCRIPTION
**Pull request by joshjms**
I added one line to trim trailing and preceding whitespace.

This is my first time contributing code, much help will be appreciated. Sorry in advance!

### CODE REVIEW

**Comment by joshjms** [comment]
image

File: cmd/program/program.go [hunk 1]
```diff
@@ -98,6 +99,9 @@ func (p *Project) CreateMainFile() error {
 		}
 	}
 
+	// Remove trailing whitespaces from ProjectName
```
Comment (Melkeydev) [inline cmd/program/program.go:102]: Since the code is self-explanatory right below, this comment is not a value add! We can remove it  :)

**Comment by Melkeydev** [comment]
This is great!
However, you have some conflicts to resolve! Ill leave those for you! Congrats on your first OS PR!

**Review by Melkeydev — CHANGES_REQUESTED** [review]
Added a comment to address the comment added + the existing conflicts

**Comment by joshjms** [comment]
Fixed! Thanks for the comment!

**Review by Melkeydev — APPROVED** [review]
LGTM!

**Pull request merged**